org.eclipse.jdt.ui.tests/chkpii/org/eclipse/jdt/ui/tests/chkpii/HTMLTidyTest.java

```diff
@@ -163,7 +164,7 @@ private boolean checkFolder(File folder) throws Exception {
 
 	private boolean checkFile(File file) throws Exception {
 		String filePath= file.getAbsolutePath();
-		String command= "tidy -eq \"" + filePath + "\"";
+		String[] command= new String[] {"tidy", "-eq", '"' + filePath + '"'};
```

Comment:
Quotes should be obsolete now when command is passed as individual arguments array  to `Runtime#exec(String[])`

Reply:
neat, it works even with space in the path, thanks :-)